### ISSUE TITLE
Update Onboarding workflow for Discussion Hosts

### ISSUE THREAD

**Issue by elletjies** [carpentries/docs.carpentries.org#778]
@acrall here is the link to the current onboarding workflow for Discussion hosts - https://docs.carpentries.org/topic_folders/instructor_development/community_discussions.html#discussion-host-onboarding

Should we discuss this during a CDT meeting?

**Comment by acrall** [comment]
Thanks @elletjies! Have added it to next week's CDT agenda for discussion

**Comment by maneesha** [comment]
This is related to #799.  When I re-organize that section, I'll follow the guidelines shared in that issue.  We can make updates to the workflow once the CDT decides on the workflow.

**Comment by maneesha** [comment]
Is this something that the @carpentries/core-team-community is still working on?

**Comment by acrall** [comment]
Thanks, @maneesha. I have added discussion host onboarding to our Asana project as a quarterly task. As part of this, I have also made a [new presentation](https://docs.google.com/presentation/d/1mal-KCfz5EaPuXXMCRiYxYsA41qPsWJfnj9ydMo9Y8Q/edit?usp=sharing) and [recording](https://youtu.be/VrWdzlD2dlE) available to the community, that is now linked on the Community Discussion Etherpad. The presentation will be reviewed annually for any necessary updates.
